File: Java/Program/Handphone.java
public class Handphone extends BarangElektronik {
    private String merek;
    private String warna;
    private int ram;
    private int storage;
    

    public Handphone(String kodeBarang, String namaBarang, int garansi, int harga, int stok, int tahunRilis,
                     String merek, String warna, int ram, int storage) {
        super(kodeBarang, namaBarang, garansi, harga, stok, tahunRilis);
        this.merek = merek;
        this.warna = warna;
        this.ram = ram;
        this.storage = storage;
    }

    public String getMerek() {
        return merek;
    }

    public void setMerek(String merek) {
        this.merek = merek;
    }

    public String getWarna() {
        return warna;
    }

    public void setWarna(String warna) {
        this.warna = warna;
    }

    public int getRam() {
        return ram;
    }

    public void setRam(int ram) {
        this.ram = ram;
    }

    public int getStorage() {
        return storage;
    }

    public void setStorage(int storage) {
        this.storage = storage;
    }
}

File: Python/Program/Handphone.py
from BarangElektronik import BarangElektronik

class Handphone(BarangElektronik):
    def __init__(self, kodeProduk, stok, harga, namaBarang, garansi, tahunRilis, merek, warna,
                 ram, storage):
        super().__init__(kodeProduk, stok, harga, namaBarang, garansi, tahunRilis)
        self.merek = merek
        self.warna = warna
        self.ram = ram
        self.storage = storage

    def get_merek(self):
        return self.merek

    def set_merek(self, merek):
        self.merek = merek

    def get_warna(self):
        return self.warna

    def set_warna(self, warna):
        self.warna = warna

    def get_ram(self):
        return self.ram

    def set_ram(self, ram):
        self.ram = ram

    def get_storage(self):
        return self.storage

    def set_storage(self, storage):
        self.storage = storage
